spec/lib/helpers/register_methods_spec.rb

```diff
@@ -506,6 +506,7 @@ module ModuleWithNoCustomPrimaryKey
     before do
       allow(ModuleThatIncludesRegisterMethods).to receive(:registry_url).and_return('http://some.registry.url')
       allow(ModuleThatIncludesRegisterMethods).to receive(:_http_get).and_return(:registry_page_html)
+      allow(ModuleThatIncludesRegisterMethods).to receive(:sleep).with(2)
```

Comment:
Ugh the test objects are bit difficult in here aren't they! With this, what about removing the `SLEEP_BEFORE_HTTP_REQ` from the test class at the top, and using `stub_const` or something in the specific context that needs it ("and SLEEP_BEFORE_HTTP_REQ is set")?

Reply:
✅